renderdoc/api/replay/replay_enums.h

```diff
@@ -1109,6 +1109,11 @@ to apply to multiple related things - see :data:`ClipDistance`, :data:`CullDista
 .. data:: OutputIndices
 
   An output containing the indices for a meshlet.
+
+.. data:: MultiViewIndex
+
+  The view being rendered to in multiview rendering. Only valid when multiview rendering is enabled.
```

Comment:
Might be worth clarifying that this is an input provided? Just to be extra clear at a glance that this isn't an output selecting a viewport.